### DISCUSSION TITLE
question for kaili.io : Run kiali.io build cmd is failed in windows os

### DISCUSSION THREAD

**Discussion by yuluo-yx** [kiali/kiali#7274] [Q&A]
### What do you want to discuss?

I want to run kiali.io website in Windows, use `make -e DORP=docker serve` cmd but it`s failed.

log:

```
Will not rebuild the image [kiali/hugo:latest].
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "C:/Program Files/Git/usr/bin/bash": stat C:/Program Files/Git/usr/bin/bash: no such file or directory: unknown.
Makefile:49: recipe for target 'serve' failed
make: *** [serve] Error 127
```

I see https://github.com/docker/cli/issues/2204 & https://stackoverflow.com/questions/7250130/how-to-stop-mingw-and-msys-from-mangling-path-names-given-at-the-command-line/34386471#34386471
but they are invalid for me.

Have solved ways?

**Comment by jmazzitelli** [comment]
_(converted to a Q&A discussion)_

I am almost 100% confident none of the Kiali make targets or scripts will work on Windows. Our development environments are targeted for Linux. I can't think of a single person that I know of that developed Kiali (or the kiali.io docs) on Windows.

To solve it - get a Linux machine :)

**Comment by yuluo-yx** [comment]
understand, I use Windows os in my local. 🥲🥲
